packages/core/src/mastra/index.ts

```
@@ -14,6 +14,7 @@ import { DatasetsManager } from '../datasets/manager.js';
 import type { MastraDeployer } from '../deployer';
 import type { IMastraEditor } from '../editor';
 import { MastraError, ErrorDomain, ErrorCategory } from '../error';
+import type { Harness } from '../harness';
```

ESLint

[error] `../harness` type import should occur after type import of `../events/types`

(import/order)